rdt-open-reader/src/main/java/io/ona/rdt/interactor/RDTJsonFormInteractor.java

```diff
@@ -38,5 +40,8 @@ protected void registerWidgets() {
         map.put(RDT_CAPTURE, new UWMalariaRDTCaptureFactory());
         map.put(COUNTDOWN_TIMER, new RDTCountdownTimerFactory());
         map.put(GPS, new RDTGpsFactory());
+        map.put(CovidConstants.Widget.UW_COVID_RDT_CAPTURE, new UWCovidRDTCaptureFactory());
```

Comment:
These statements should not be committed, they're only meant to be local changes when running `JMAG`.